source/_posts/2018-09-30-hacktoberfest.markdown

```diff
@@ -0,0 +1,56 @@
+
+Hacktoberfest will create a lot of work us. In 2017 we processed 625 Pull Requests in our three main repositories and some more for other parts of Home Assistant during October. We still only have limited resources and for this edition of Hacktoberfest, so we decided to set some rules. These rules will help us and keep your frustration level to a minimum.
+
+**We want to focus on new contributors and people who want to get started on working on an Open Source project.**
+
+## {% linkable_title Configuration %} General
+
+- Fill out the Pull Request template.
+- Check our code (lint and unit tests) before you submit your Pull Request.
+- Pull requests which are not passing the CI usually get less or no attention.
+- There is no guarantee that your Pull Request is processed during October. Sorry.
+- We reserve the right to close Pull Requests with missing parts (see the Checklist in the Pull Request template), insufficient data or no response after they were submitted. The Pull Requests can always be re-opened later when you are ready or all elements are in place.
+
+## {% linkable_title Home Assistant %}
+
+- Get familiar with our [Style Guidelines](https://developers.home-assistant.io/docs/en/development_guidelines.html) and the existing code.
+- Go through the [Developer documentation](https://developers.home-assistant.io).
+- Keep your Pull requests small. One change per Pull request is faster to review than 19 changes. The same applies to new integrations. Start with the component and one platform. Add the other platforms later as this would get you feedback which you can incorporate in the coming platforms.
+- Don’t forget the documentation. New integrations need documentation, and changes often lead to updates to documentation as well. You know how it works, but the users don’t.
+- Architectural changes need to be discussed before a change is proposed. Also, keep in mind that those kinds of Pull Request may not be reviewed during October.
+- If you update a dependency, then you need to make sure that it will work. This means, provide the configuration sample and other details (used devices, firmware version, etc.).
+
+## {% linkable_title Home Assistant Documentation %}
+
+- Make yourself familiar with [Markdown](https://developers.home-assistant.io/docs/en/documentation_index.html).
+- Understand our [Standards and Guidelines](https://developers.home-assistant.io/docs/en/documentation_standards.html).
+- For small changes like typos and clarifications, use the GitHub editor functionality. 
+- For more significant changes, we highly recommended that you set up an environment to build the documentation locally before opening a Pull Request.
+- Keep an eye on the branches. `current` is for changes on the existing documentation, `next` for new features or integrations that will be available with the next release.
+
```

Comment:
IMHO we are short on front-end developers, I would definitely recommend listing the Polymer/frontend repo because of that.

Reply:
Set our focus on others than main.